Recipe Detail/recipe.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="FoodApp is the first Application for food delivery for Gaza Restaurants, ... FoodApp is Palestinian foodies favorite food ordering app and website"
+    />
+    <title>Recipe Detail</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header >
+      <section class="header-content">
+        <a href="Community/community.html" class="pre-page"
+          ><img src="media/chevron-circle-left.png" alt="previous page"
```

Comment:
Link ***all images*** correctly You can put this code until it becomes correct
`<img src="../media/Page 2/<name the img>" alt="previous page"`